src/pc/1.9/chunk.js

```diff
@@ -341,10 +361,10 @@ class Chunk {
         } = this.readSection(chunk.slice(offset), skyLightSent)
 
         offset += size
-        blocksAddition = this.eatPackedBlockLongs(value.dataArray, value.palette, value.bitsPerBlock)
-        blocklightsAddition = value.blockLight
+        blocksAddition = this.eatPackedBlockLongs(value.data_array, value.palette, value.bits_per_block)
+        blocklightsAddition = value.block_light
 
-        skylightsAddition = skyLightSent ? value.skyLight : Buffer.alloc(skyLightSize)
+        skylightsAddition = skyLightSent ? value.sky_light : Buffer.alloc(skyLightSize)
```

Comment:
snake_case is wrong

Reply:
ProtoDef compiler enforces it

Reply:
it should transform the names to camelCase then